### PR TITLE
add dynamic vm to db backup test

### PR DESCRIPTION
**Pull request by kzvyahin**
Add dynamic vm for db backup (nfs, smb)
{{pytest: cfme/tests/configure/test_db_backup_schedule.py}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 4227e2cd6506995c40c1c58c13c475ea2792ec00:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: cfme/tests/configure/test_db_backup_schedule.py [hunk 1]
```diff
@@ -146,7 +170,7 @@ def test_db_backup_schedule(request, db_backup_data):
     # the dash is there to make strftime not use a leading zero
     hour = dt.strftime('%-H')
     minute = dt.strftime('%-M')
-
+    hostname = db_depot_machine_ip + db_backup_data.sub_folder
```
Comment (jkrocil) [inline cfme/tests/configure/test_db_backup_schedule.py]: We should no longer call this `hostname`. It should be `uri` or `url` instead.
Subsequently, the proper way to join URL components would be:
```
import urlparse
urlparse.urljoin("192.168.0.1", "/my_subfolder")
```

**Comment by dajoRH** [comment]
Lint report for commit 16a989d2ed51c2f6758d521d77d3d4b759d8ba8a:

`cfme/tests/configure/test_db_backup_schedule.py`:
- :red_circle: Line 11:1: **F401** *'urljoin' imported but unused*

Please, rectify these issues :smirk: .

Commit inspection report:

**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 6b30c9f9207c2b14a6475601efe0a4c9804291a0:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Pull request merged**